storage/libradb/src/ledger_store/mod.rs

```diff
@@ -123,6 +114,17 @@ impl LedgerStore {
                 )
                 .into());
             }
+            if ledger_info_with_sigs
+                .ledger_info()
+                .next_validator_set()
+                .is_none()
+            {
+                return Err((LibraDbError::NotFound(format!(
+                    "Epoch {} change proof",
+                    ledger_info_with_sigs.ledger_info().epoch()
+                )))
```

Comment:
This message isn't accurate. It's not `NotFound`. The error is either because the DB is corrupted, or `end_epoch` is too new so the iterator reached the latest ledger info.

Reply:
hmm, I didn't realize `NotFound` is for db corruption